.ci/acceptance-tests/ansible-integration.sh

```diff
@@ -8,26 +8,27 @@ echo "${ANSIBLE_TEMPLATE}" > /tmp/ansible-template.ini
 set -e
 set -x
 
-# Get the newest version of Ansible from the PR
+# Install ansible from source
+git clone https://github.com/ansible/ansible.git
```

Comment:
In the new collections world will we have our own release cadence out of step with Ansible core? i.e., should we pin this to the latest release rather than head? (I kinda hope not, because unless there's an automated way to get the latest version number it'll be a huge PITA)